ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java

```diff
@@ -100,9 +102,18 @@ protected PubsubMessage processElement(PubsubMessage message)
           zip(GENERIC_URI_SUFFIX_ELEMENTS, uri.substring(GENERIC_URI_PREFIX.length()).split("/")));
     } else if (uri.startsWith(StubUri.PREFIX)) {
       payload = StubUri.parse(uri, attributes);
+
+      if (attributes.get("messageId") != null) {
```

Comment:
We've been using snake_case for attributes, so I'd prefer we use `"message_id"`. Also, can you factor this string out to an `Attributes.MESSAGE_ID` constant?